### DISCUSSION TITLE
Contributions don't show up in the new account.

### DISCUSSION THREAD

**Discussion by sathwik-aileneni** [community/community#53496] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I had a GitHub account for work (username:account_1) and I used to have a single email in that (email@work.com). I have another account for personal stuff (username:account_2). My contributions graph in the work account is pretty high, Recently because of some personal preference, I moved that email (email@work.com) to my personal account and deleted the account_1. All my commits in my organization repositories are changed from "committed by account_1" to "committed by account_2" automatically (Good). In my personal account, I also changed the settings (Show contributions from private repositories) to true. But my contribution graph is still not updated in account_2. I found online that we need to wait for 24 hrs so that GitHub updates it. But it's been 2 days and still, the graph is not updated. Do I need to wait a couple of days more or is there something that I am missing? Any help will be appreciated.

**Comment by NJacksonDev** [comment]
Here is a helpful article from GitHub about missing contributions. There is also a virtual agent (same page as article) that Github provides which may be of help - hopefully your issue is resolved soon!

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Reply by sathwik-aileneni** [reply]
Thank you. Tried this, But the VA is also not very clear on my specific problem. I created a support ticket.

**Comment by sathwik-aileneni** [comment]
Looks like there is no issue at my end, The 24-hour cycle is a bit delayed it seems.
